.mergify.yml

```diff
@@ -1,67 +1,29 @@
 pull_request_rules:
   - name: Merge PRs that are ready
     conditions:
-      - status-success=Travis CI - Pull Request
+      - check-success=Check Code Style
+      - check-success=Check Docs
+      - check-success=Check Code Compilation
+      - check-success=Run documentation tests
+      - check-success=Build and Test (Scala 2.12 & JDK 8)
+      - check-success=Build and Test (Scala 2.12 & JDK 8)
+      - check-success=Build and Test (Scala 2.13 & JDK 11)
+      - check-success=Build and Test (Scala 2.13 & JDK 11)
+      - check-success=MultiJvm Test (Scala 2.12 & JDK 8)
+      - check-success=MultiJvm Test (Scala 2.12 & JDK 8)
+      - check-success=MultiJvm Test (Scala 2.13 & JDK 11)
+      - check-success=MultiJvm Test (Scala 2.13 & JDK 11)
```

Comment:
If I right understand, should be
```suggestion
      - check-success=Build and Test (Scala 2.12 & JDK 8)
      - check-success=Build and Test (Scala 2.13 & JDK 8)
      - check-success=Build and Test (Scala 2.12 & JDK 11)
      - check-success=Build and Test (Scala 2.13 & JDK 11)
      - check-success=MultiJvm Test (Scala 2.12 & JDK 8)
      - check-success=MultiJvm Test (Scala 2.13 & JDK 8)
      - check-success=MultiJvm Test (Scala 2.12 & JDK 11)
      - check-success=MultiJvm Test (Scala 2.13 & JDK 11)
```

Reply:
You are right. I move this around so many times and at the end I messed it up.